CONTRIBUTING.md

```diff
@@ -2,51 +2,59 @@
 We want to make contributing to this project as easy and transparent as
 possible.
 
-## Pull Requests
-We actively welcome your pull requests.
-
-1. Fork the repo and create your branch from `master`.
-2. If you've added code that should be tested, add tests.
-3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes.
-5. Make sure your code lints.
-6. If you haven't already, complete the Contributor License Agreement ("CLA").
+## Developer workflow
+- Active development should happen on your fork of the repositories
+- Name your PR in a way that unambiguously identifies the feature or fix
+- Follow the contribution guide to ensure code is conformant with conventions and style
+- Try to make small, logically independent, self-documenting commits (and reflect this in the commit messages by providing brief rationale/change summary). Feel free to amend and force push changes to your PR if necessary.
+- Encouraging to create draft PRs to gather early feedback
+- Request reviews from at least one Habitat core team member (if the scope of changes necessitates this, two or more reviewers is recommended)
+- We have adopted squash-and-merge as the policy for incorporating PRs into the master branch.  We encourage more smaller/focused PRs rather than big PRs with many independent changes.  This also enables more velocity by merging PRs into master quickly and reducing the need to rebase due to changes on master.
```

Comment:
```suggestion
- We have adopted squash-and-merge as the policy for incorporating PRs into the master branch.  We encourage more smaller/focused PRs rather than big PRs with many independent changes.  This also enables faster development by merging PRs into master quickly and reducing the need to rebase due to changes on master.
```